### ISSUE TITLE
"Open API Spec" doesn't appear to work

### ISSUE THREAD

**Issue by DanaEpp** [usebruno/bruno#1986]
## Issue
I am trying to import a sample API spec doc (OAS 3.0.1) using the "Open API Spec" link in the UI. 

![image](https://github.com/usebruno/bruno/assets/11337016/1358563b-057f-4048-8468-8cafbe9ef662)

[The spec doc I am using can be found here](https://github.com/OWASP/crAPI/blob/develop/openapi-spec/crapi-openapi-spec.json). 

While I can select it through the file explorer, it doesn't do anything. It doesn't load up, and I see no errors. 

This is for Golden Edition v1.12.3.

**Comment by sanjai0py** [comment]
Hi @DanaEpp, I'm sorry to say that at the moment Bruno only allows for importing YAML specifications of OpenAPI. However, we are working on fixing this. For now, you can create an empty API specification file and then simply copy and paste the JSON values into it. I've attached a demo video for reference. If you have any further questions or if there's anything else I can help you with, please let me know!

https://github.com/usebruno/bruno/assets/84461672/2cb27e11-67b2-49b3-88c3-baa050ccc87f

**Comment by DanaEpp** [comment]
I've heard on Discord that the issue here is that the API spec import only supports YAML format currently, and not JSON.

So, this should be marked as a Feature Enhancement—a pretty major one, if you ask me—but still just an enhancement.

**Comment by Gitznik** [comment]
I managed to import json openapi specs once on `v1.12.2`, but except that one occasion, every time I select the `openapi.json` file, a location and then nothing happens when I click `import`.

**Comment by melroy89** [comment]
yup. I noticed that JSON isn't working, but YAML is. So this is a missing feature IMO.

**Comment by NiklasBr** [comment]
> JSON isn't working, but YAML is

A hacky suggestion: According to spec a valid JSON file should also be valid YAML, so try to change the file extension to `.yaml` and open again.

**Comment by melroy89** [comment]
Ahha no. Json isn't yaml. It's a totally different format.

**Comment by NiklasBr** [comment]
[JSON is valid YAML](https://til.codeinthehole.com/posts/json-is-valid-yaml/)

**Comment by melroy89** [comment]
> [JSON is valid YAML](https://til.codeinthehole.com/posts/json-is-valid-yaml/)

Ow... OK. Today I learned that json is mostly valid yaml.

**Comment by anusree-bruno** [comment]
This issue has been resolved - you can now open the spec in the app with JSON extension.
